prg_test_helpers.go

```diff
@@ -0,0 +1,53 @@
+package test
+
+import (
+	"crypto/rand"
+	"encoding/binary"
+	"testing"
+
+	. "github.com/bjartek/overflow"
+	"github.com/onflow/cadence"
+	"github.com/stretchr/testify/require"
+)
+
+// calls a script which creates a new PRG resource from the given seed and
+// random salt, gets the next uint64 and destroys the prg resource before
+// returning the next uint64
+func GetNextUInt64NewPRG(
+	o *OverflowState,
+	t *testing.T,
+	seed []byte,
+) (uint64, error) {
+
+	randResult := o.Script(
```

Comment:
I had a quick look at the PRG contract. 
Just a heads-up we may need to make some interface changes that can impact this test (for instance the size of the seed and salt). Not sure if we need to merge this PR first, or wait for merging the PRG contract first.

Reply:
We might as well make those changes to the PRG contract first. Do we have a reference interface design or would it be most helpful for me to get a PR going with the PRG contract to communicate feedback?

Reply:
No there isn't a reference design for this specific case unfortunately. Xorshift128+ works with a seed of 16 bytes. In our case there are 32 bytes of seed and 8 bytes of salt. Apologies I didn't give you an early heads-up about it. 
I'm still thinking of the safest way to funnel those into 16 bytes, that preserves all the properties.

I think a draft PR with the PRG contract may be the best way to continue the discussion.

Reply:
No worries, thanks for clarifying - PR https://github.com/onflow/random-coin-toss/pull/6 should give us a diff to communicate against the current implementation